Added pages for "Contact" and "About" (#128)
mumble-voip/mumble-www: hugo/content/about.md

---
title: About

> 

Reviewer: Remove trailing empty line in front matter
```suggestion
```

---

Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.

## Source Code & Development

The source code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).

> For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip/).

Reviewer: GitHub itself links to org without slash, so we should use that. Strangely enough slash does not redirect to without.

```suggestion
For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip).
```

## Licenses

Mumble is Free and Open-Source Software.
The core project is licensed under a BSD-style license. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.

> Our other projects may use different licenses. For details we refer to the respective GitHub repositories, see: [Overview of all Mumble projects on GitHub](https://github.com/mumble-voip/).

Reviewer: Is this a typo or intended this way?

```suggestion
Our other projects may use different licenses. For details please refer to the respective GitHub repositories, see: [Overview of all Mumble projects on GitHub](https://github.com/mumble-voip/).
```

Reviewer 2: It was intended this way, but I think your suggestions is better

Author: Agreed.

---
title: About

---

Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.

## Source Code & Development

> The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).

Reviewer 2: ```suggestion
The source code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
```

> For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).

Reviewer 2: ```suggestion
For other Mumble Projects, see the [overview of repositories on GitHub](https://github.com/mumble-voip/).
```

## Licenses

Mumble is Free and Open-Source Software.

> The core project is licensed under a BSD-style License. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.

Reviewer 2: ```suggestion
The core project is licensed under a BSD-style license. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
```

> There are additional licenses, you can find them in the GitHub repositories, see: [Overview of all Mumble Projects](https://github.com/mumble-voip/)

Reviewer 2: ```suggestion
Our other projects may use a different license. For details we refer to the respective [GitHub repositories](https://github.com/mumble-voip/)).
```

# About us

We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.

## Core Team

Active Team Members are:

> | Name | Github Profile | Assignment |

Reviewer 2: ```suggestion
| Name | GitHub Profile | Assignment |
```

---
title: About

> toc: true

Reviewer: Declares that it enables a toc, but I do not see any when rendering the page??

```suggestion
```

---

> 
> # About the project

Reviewer: Double headline

![image](https://user-images.githubusercontent.com/93181/107993305-964b4100-6fda-11eb-919d-551041190a51.png)

```suggestion
```

Author: Ok, will change.

Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.

## Source Code & Development

The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).

For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).

## Licenses

> Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).

Reviewer: Suggestion: We already link FOSS above. I would drop it here.

IMO linking the word _License_ here does not read fluent. It would be better, but still not optimal, to link _a BSD-style License_. I would add another sentence to signify that we are linking to the LICENSE/LICENSE file specifically here.

```suggestion
Mumble is Free and Open-Source Software and licensed under [a BSD-style License](https://www.mumble.info/LICENSE).
```

```suggestion
Mumble is Free and Open-Source Software and licensed under a BSD-style License. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
```

Side note: We should probably mention our varying used licenses here, not just the project source code, and provide a reasonable intro to that. But this is fine for a starter.

Author: > IMO linking the word License here does not read fluent. It would be better, but still not optimal, to link a BSD-style License. I would add another sentence to signify that we are linking to the LICENSE/LICENSE file specifically here.

I agree, so I will use your second suggestion, correct?

> Side note: We should probably mention our varying used licenses here, not just the project source code, and provide a reasonable intro to that. But this is fine for a starter.

It is mentioned below, but it can be improved I guess.

Reviewer: Yes

> Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a custom BSD-style [License](https://www.mumble.info/LICENSE).

Author: If thats the case, I can say that it is a BSD-license.
I read somewhere that it is costum and only inspired by bsd licenses, but I can't find the source now.

> Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a custom BSD-style [License](https://www.mumble.info/LICENSE).

Reviewer 2: Comparing https://github.com/mumble-voip/mumble/blob/master/LICENSE to https://opensource.org/licenses/BSD-3-Clause shows that in our license file there are a few extra lines mentioning the list of contributors. But as far as I can tell this doesn't affect the legal stuff so in that regards it should be the same as the BSD-3-Clause license

All Licenses are also listed on GitHub:

- [Mumble Repo (Client & Server)](https://github.com/mumble-voip/mumble/)
    - [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE)

> - [Other Mumble Projects (Overview)](https://github.com/mumble-voip/)

Author: Maybe, but I will rewrite it a bit.

I will list the main license and a link to the general github overview and tell people to look at the repos for further licenses.

# About us

> We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.

Author: I don't see a reason for that.

> We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.

Reviewer 2: Afaik this is the usual way to use this kind of formulation (not 100% sure though) :shrug:

> We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.

Author: Never seen that, a quick search shows that there are many examples of "both professional and non-professional", so I don't think the ":" are necessary.

> | Name: | Github Profile: | Assignment: |

Author: I think colons look better, but if you don't want to have them.

## Core Team

- Davide Beatrici [davidebeatrici](https://github.com/davidebeatrici) - Core developer
- Robert Adam [Krzmbrzl](https://github.com/Krzmbrzl)- Core developer, GitHub Management
- Nik Reist [ZeroAbility](https://github.com/ZeroAbility) - Release Management

> - Jan Klass [Kissaki](https://github.com/Kissaki) - Community work

Author: Honestly I think that is a bit too much "on GitHub".
People will directly see where it goes to, so there is no need imo for clarifying that.

> - Jan Klass [Kissaki](https://github.com/Kissaki) - Community work

Reviewer 2: They will only see it when hovering over the link. And without doing that having these kind of names simply floating there without context is pretty confusing imo

> - Jan Klass [Kissaki](https://github.com/Kissaki) - Community work

Author: What about a table instead?
With that I could mention Github just once and it would also look nice and clean.

> - Jan Klass [Kissaki](https://github.com/Kissaki) - Community work

Reviewer 2: I'd be fine with that as well

> | Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |

Reviewer 2: I feel this should represents the current situation (that's also why Thorvald isn't listed at all).
Nonetheless you are free to suggest a different role description here. This was just my original suggestion and since you did not answer on my ping on this topic, we left it at that :shrug:

> | Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |

Author: @Krzmbrzl Could you maybe talk about this in one of your team meetings, so that we could have an accurate representation of every team member?
@Kissaki What would you like as a description?

> | Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |

Reviewer: Whatever

> | Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |

Author: "Whatever" is not really a good answer. Maybe other team members want a different description etc. as well. I will wait for the results of your team meeting, thus will unresolve this.

- [Natenom](https://github.com/Natenom) - Testing

## Contributors

> We are also supported by contributors from the community.
> You find a [List of Contributors on Github](https://github.com/mumble-voip/mumble/graphs/contributors).

Reviewer 2: ```suggestion
Without further contributors from within the community, this project would not be possible.
You  can find a [list of contributors on GitHub](https://github.com/mumble-voip/mumble/graphs/contributors).
```

Author: I see your intention, but I think we might reformulate that a bit.
Something like: "Contributions from the community are an important part of the project."

I know that's not perfect either, but thats more the direction I would go, for two reasons:
1. language (sounds rather german, no offense)
2. exaggeration (main contributors are from the team and former team; so community is a nice addition, but not like 50% of the project)

Author: By the way: Should I also mention honorable former team members or a special thanks?

Reviewer 2: > language (sounds rather german, no offense)

I'm actually pretty sure that this is a fine english sentence :shrug: .

But I agree: Your formulation is probably better as it is more correct :+1: 

> By the way: Should I also mention honorable former team members or a special thanks?

Nah, I wouldn't do that. People will find out about those when looking at the contributor page on GitHub.

## Contact

> Take a look at our [Contact page](https://www.mumble.info/contact).

Reviewer: This should be a relative hugo native link.

Author: I don't like it, but I will add it anyway.

Reviewer: Do you have a reason for disliking it?

Not using native, relative links leads to technical issues if you make changes to how and where you deploy.

Author: I like and understand the concept of relative linking; what I dislike is described here: #135.
In the meantime I already added relref links in my offline version, which I will upload along with all other changes, once it's ready.

# Security notes

> Our [Security page](https://www.mumble.info/security) contains information on security issues, support & contact.

Reviewer: Should be a relative, hugo native link.

# Privacy notes

> Our Privacy policy can be found [here](https://www.mumble.info/privacy).

Reviewer: Should be a relative, hugo native link.

For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).

## Licenses

Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).

All Licenses are also listed in the GitHub repositories, see: [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE) & [Overview of other Mumble Projects](https://github.com/mumble-voip/)

# About us

We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.

## Core Team

Active Team Members are:

| Name: | Github Profile: | Assignment: |
| --- | --- | --- |
| Davide Beatrici | [davidebeatrici](https://github.com/davidebeatrici) | Core developer |
| Robert Adam | [Krzmbrzl](https://github.com/Krzmbrzl) | Core developer, GitHub Management |
| Nik Reist | [ZeroAbility](https://github.com/ZeroAbility) | Release Management |
| Jan Klass | [Kissaki](https://github.com/Kissaki) | Community work |

> | Natenom | [Natenom](https://github.com/Natenom) | Testing |

Author: Example table.

Reviewer 2: In case of Natenom I would leave the real name entry blank.

Author: :thinking: I think leaving an empty space is not so good...
How about clarifying that it is a nickname?
Like `Natenom (Nickname)`

Reviewer 2: If you don't want to leave it blank, fill the table cell with a single `-` to symbolize that the real name is not provided.

Author: Well, if you want it like that.

---
title: About
toc: true
---

# About the project

Mumble is a free, open source, low latency, high quality voice chat application.

## Source Code & Development

You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).

## Licenses

<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->

> <!-- more licenses? should be as correct and complete as possible -->

Reviewer 2: I think we don't need to specify licenses here explicitly. If you think it's important you can state something like "We are generally using Open Source license without copy-left clauses (e.g. BSD-3)" but in the end the user can simply checkout the license for the individual projects.

Author: Then I will link to Github and tell people to look at the licenses of each project.
Ok?

Reviewer 2: Sounds good

# Who we are

<!-- Title could also be: Developers, Team etc. -->

<!-- improve -->

We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.

<!--
Link to Github Team page:
https://github.com/orgs/mumble-voip/people

> -->

Reviewer 2: Probably a good idea :+1: 

The list is effectively outdated though (in terms of who is still active) since the majority of people listed there have not contributed to the project since I started working on it.

Author: > The list is effectively outdated though

Thats a problem.
If we list the Core team, the information should be fairly recent imo.

Reviewer 2: I would simply not list that. I think the contributors page on GitHub contains all information that a user might want to know. Thus imo it should be enough to link to that :shrug:

Author: I disagree:
1. The Github Graph shows some inactive team members first.
2. One of the main intentions of an about page is to inform users, who the people behind the project are and this would be done best by listing the core team.

But if you don't want that, there is of course nothing I can do :rabbit:.

Reviewer 2: Fair enough. In that case I think we could list
- @davidebeatrici - Core developer
- @Krzmbrzl - Core developer, GitHub Management
- @ZeroAbility - Release Management
- @Kissaki  - Community work
- @Natenom - Testing

I think as far as the core team and their current main tasks are concerned, that should be it (correct me if that's not the case).

Author: I assume you want no real names (though they can be found out easily anyway) ?
I will link to the Github profiles then.

> correct me if that's not the case

I have no clue :shrug:. But doesn't Kissaki & ZeroAbility do other things as well?

Reviewer 2: If the names are available on the respective GitHub profile pages (which should be linked in any case), I think it is fine to also list the real names on the Website. If the real name is only contained in the commits, I would say that they should not be mentioned on the website.

> I have no clue shrug. But doesn't Kissaki & ZeroAbility do other things as well?

This comment was actually more directed towards the mentioned team members. I didn't expect you to know that :grin:

Author: > If the names are available on the respective GitHub profile pages (which should be linked in any case), I think it is fine to also list the real names on the Website

Ok, sounds good. Real names are more trustworthy.

> This comment was actually more directed towards the mentioned team members. I didn't expect you to know that

I already assumed that. But because they do respond very rarely (which is to some degree understandable if you got much to do and get many messages), it is probably on us to figure out the situation.

Reviewer 2: > it is probably on us to figure out the situation.

Indeed. That's why I'd suggest overtaking the descriptions I provided until someone comes around and complains :D

> <!-- Distinction between active and non-active developers? -->

Reviewer 2: I'd rather not do that since this is very prone to become outdated. You could link to https://github.com/mumble-voip/mumble/graphs/contributors though. There the user can filter for the time interval they are interested in and then evaluate for themselves who was active in that period.

Author: Well that is not really ideal.
See above :point_up:.

> <!-- could contain more blabla about vision, free-software dedication etc. -->

Reviewer 2: No real need to do that. We could instead link to a good website explaining what Open Source is about.

Author: Well, [wikipedia](https://en.wikipedia.org/wiki/Open-source_model) does not seem to be ideal...
if you know a good website.

Otherwise I will simply leave this out.

Reviewer 2: I think https://en.wikipedia.org/wiki/Open-source-software_movement is not too bad for this purpose :thinking:

Author: What about calling Mumble `Free Software` and link to the [Wikipedia article](https://en.wikipedia.org/wiki/Free_software)?

Reviewer 2: To be really correct, Mumble is FOSS (Free and Open Source Software). That basically covers everything. Thus https://en.wikipedia.org/wiki/Free_and_open-source_software is probably the most correct thing to link here :thinking:

Author: I just saw that you already describe mumble as "Mumble is a free, open source, low latency, high quality voice chat application.", so how should I include it?
For now I have only mentioned it in the section about "Licenses".

Reviewer 2: You could simply turn "free, open source" into a link to that Wikipedia page :thinking:

> <!-- Credits/List of Contributors ? -->

Reviewer 2: Not here. The user may again be referred to https://github.com/mumble-voip/mumble/graphs/contributors

> <!-- https://www.mumble.info/AUTHORS -->

Reviewer 2: I would not link to that since it is probably outdated and furthermore it is exposing their mail addresses. Linking that on a website will just attract web crawlers.
Besides: All contributors can be found in the GitHub stats (or `git log` if someone really wants to know it)

Author: Ok, will use the stats/graph then.

<!-- see also: "contribute" below -->

# About the website

<!-- link to github repo 

https://github.com/mumble-voip/mumble-www
-->

<!-- information on who operates the website etc. -->

<!-- add category? 

> # Contribute

Reviewer 2: Having this section on the `About` page seems a bit odd anyways :thinking:

Author: You are right, we have the general contribute page anyways, so it's visible on first sight.
I was just looking at other websites and also found this.
